src-java/rule-manager/rule-manager-implementation/src/main/java/org/openkilda/rulemanager/Constants.java

```diff
@@ -130,6 +130,8 @@ public static final class Priority {
         public static final int SERVER_42_INGRESS_DOUBLE_VLAN_FLOW_PRIORITY = FLOW_PRIORITY + 10;
         public static final int SERVER_42_INGRESS_DEFAULT_FLOW_PRIORITY = FLOW_PRIORITY - 10;
         public static final int SERVER_42_PRE_INGRESS_FLOW_PRIORITY = FLOW_PRIORITY;
+
+        public static final int PING_INPUT_PRIORITY = ISL_EGRESS_VLAN_RULE_PRIORITY_MULTITABLE + 10;
```

Comment:
My bad. It should be +1. 

Let me explain. We have 3 ISL rules
1. VXLAN which sends packets from isl to egress
2. VLXAN which sends packets from isl to transit
3. vlan which sends packets from isl to transit.

They have very strict priorities:
https://github.com/telstra/open-kilda/blob/4db6d47f97df98c80c62227997eded0284763cef/docs/design/multi-table-pipelines/Table_0.png

We have a note that priorities must be 
VXLAN egress > VXLAN transit > vlan


Now they are:
```
        public static final int ISL_EGRESS_VXLAN_RULE_PRIORITY_MULTITABLE = FLOW_PRIORITY - 2;
        public static final int ISL_TRANSIT_VXLAN_RULE_PRIORITY_MULTITABLE = FLOW_PRIORITY - 3;
        public static final int ISL_EGRESS_VLAN_RULE_PRIORITY_MULTITABLE = FLOW_PRIORITY - 5;
        ```
        
        So the only priority we can set is `FLOW_PRIORITY - 4` (which is equal to `ISL_EGRESS_VLAN_RULE_PRIORITY_MULTITABLE + 1` actually). 
        
        So could you please set it back to `ISL_EGRESS_VLAN_RULE_PRIORITY_MULTITABLE + 1`  and move it to line 82 (near with `ISL_EGRESS_VLAN_RULE_PRIORITY_MULTITABLE`)

Reply:
done